lib/src/common/model/repos_releases.dart

```diff
@@ -30,7 +30,7 @@ class Release {
 
   /// Target Commit
   @JsonKey(name: "target_commitish")
-  String targetCommitsh;
+  String targetCommitish;
```

Comment:
This would be easy to not break people, however unlikely. Could you add something like this?
```
  /// Target Commit (Deprecated)
  ///
  /// `targetCommitsh` was a typo in previous versions and has been corrected.
  /// Use `targetCommitish` instead.
  /// This will be removed in the next major release
  @deprecated
  String get targetCommitsh => targetCommitish;
```

Reply:
On second thought, I'm going to get this rename in in a smaller PR as 5.2.1

Reply:
This fix has been merged to master in https://github.com/DirectMyFile/github.dart/pull/169
If you merge origin master, this diff will go away.